relation/index.js

```diff
@@ -108,7 +110,25 @@ module.exports = yeoman.generators.Base.extend({
           }
           return m;
         },
-        validate: validateName
+        validate: function(value) {
+
+          var done = this.async();
+
+          if (validateName(value) !== true) {
+            done(validateName(value));
+          } 
+
+          modelDef.properties(function(err, list) {
+            if (err) return done(err);
```

Comment:
Since the `done` callback expects a single argument with a boolean value, I think we should handle the error different way here. Something like

```js
if (err) {
  console.log('Cannot validate relation name against existing model properties:', err.message);
  // Let the user add this relation even if we cannot confirm there is no naming clash
  return done(true);
}
```